lib/auth/webauthncli/api.go

```diff
@@ -67,6 +68,17 @@ func Login(
 	ctx context.Context,
 	origin string, assertion *wanlib.CredentialAssertion, prompt LoginPrompt, opts *LoginOpts,
 ) (*proto.MFAAuthenticateResponse, string, error) {
+	// origin vs RPID sanity check.
+	// Doesn't necessarily means a failure, but it's likely to be one.
+	switch rpID := assertion.Response.RelyingPartyID; {
+	case origin == "", assertion == nil: // let downstream handle empty/nil
```

Comment:
The case is assuming that `assertion == nil` is a valid case: 
> // let downstream handle empty/nil

but `switch rpID := assertion.Response.RelyingPartyID` always references to `assertion.Response` and in case of `assertion == nil` will case a panic.

Reply:
Good catch  🤦 

Since this is a backport, I'll open a fix on master and pull it here.

Reply:
Fixed, see also #13761.